<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pdf_page_selector</class>
 <widget class="QWidget" name="pdf_page_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="start_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Page</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_page_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="end_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>End Page</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_page_edit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_bt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_bt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="start_page_view">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>151</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_page_view">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>151</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
